GHzDACs/jump_table.py

```diff
@@ -78,6 +90,7 @@ def __str__(self):
         return "%s %d cycles" % (self.NAME, self.cycles)
 
     def as_int(self):
+        """See Operation.as_int."""
```

Comment:
Might it be better to leave off the docstring if we're not going to overwrite it? I don't know.